src/services/gasprice/gasprice.service.js

```diff
@@ -1,21 +1,36 @@
-// Initializes the `gasprice` service on path `/gasprice`
-const createService = require('feathers-memory');
-const hooks = require('./gasprice.hooks');
+const DEFAULT_PRICE = 100; // ethGasStation returns price in 10 gweis. 100 is really 10 gwei
 
 module.exports = function gasPrice() {
   const app = this;
-  const paginate = app.get('paginate');
 
-  const options = {
-    name: 'gasprice',
-    paginate,
+  const gasPriceService = {
+    async find() {
```

Comment:
If there is no usage for this service we can close it too, since we are working on network with constant gas price